ReadMe.md

```diff
@@ -84,6 +84,11 @@ GLOBAL OPTIONS:
    --config FILE, -c FILE  Config FILE name. (default: "/etc/clickhouse-backup/config.yml") [$CLICKHOUSE_BACKUP_CONFIG]
    --help, -h              show help
    --version, -v           print the version
+
```

Comment:
oops, sorry I forget 
`clickhouse-backup help create`
should show --partitions information

Reply:
> 

It can show by default.